src/utils/ServiceRequest.ts

```diff
@@ -95,18 +95,16 @@ export class ServiceRequest<T> {
       }
     }
 
-    Logger.debug(`Request:`, this.logContext, JSON.stringify(requestWithoutAuth, null, 2));
-    this.query = queryParams(request, this.logContext);
-
     try {
       this.body = request.body.json();
     } catch (exception) {
-      this.error = {
-        errorMessage: `${this.logContext.getBaseScope()}: No valid JSON request for ${this.logContext.getFormattedScopeString()}`,
-      };
-      this.success = false;
-      return;
+      Logger.info("No JSON body found", this.logContext);
```

Comment:
Looks like we're changing the behaviour from having no body being a critical failure to not caring, can you explain (maybe in a comment) why it's okay not to care?

Reply:
The endpoints validate the properties they expect in the body. If the request did not pass a valid JSON body we have an empty body object and the endpoints typically return 400